src/reason-parser/lexer_warning.ml-4.07

```diff
@@ -0,0 +1,3 @@
+let warn_latin1 lexbuf =
```

Comment:
They changed this again in 4.07?

Reply:
No change vs 4.06. This is just a copy of the 4.06 file as a quick hack to allow the build to complete with minimal changes. `select.ml` should eventually be updated to handle this rather than duplicating this code for every new version of OCaml. I unfortunately haven't had time to make such an update.